spec/features/professors/supervisions/supervisions_index_spec.rb

```diff
@@ -7,46 +7,70 @@
     login_as(professor, scope: :professor)
   end
 
-  describe '#index', js: true do
-    context 'when shows all the supervisions of tcc one calendar' do
-      it 'shows all the supervisions of tcc one with options' do
-        orientation = create(:current_orientation_tcc_one)
-        orientation.professor_supervisors << professor
+  context 'when shows all the supervisions of tcc one calendar' do
+    let(:orientation) { create(:current_orientation_tcc_one) }
+    let(:index_url) { professors_supervisions_tcc_one_path }
 
-        index_url = professors_supervisions_tcc_one_path
-        visit index_url
-
-        expect(page).to have_content(orientation.short_title)
-        expect(page).to have_content(orientation.advisor.name)
-        expect(page).to have_link(orientation.academic.name,
-                                  href: professors_supervision_path(orientation))
+    before do
+      orientation.professor_supervisors << professor
+      visit index_url
+    end
 
-        orientation.calendars.each do |calendar|
-          expect(page).to have_content(calendar.year_with_semester_and_tcc)
-        end
+    it 'shows the orientation information' do
+      expect(page).to have_content(orientation.short_title)
+      expect(page).to have_content(orientation.advisor.name)
+      expect(page).to have_link(
+        orientation.academic.name,
+        href: professors_supervision_path(orientation)
+      )
+    end
 
-        expect(page).to have_selector("a[href='#{index_url}'].active")
+    it 'shows the calendar information' do
+      orientation.calendars.each do |calendar|
+        expect(page).to have_content(calendar.year_with_semester_and_tcc)
       end
     end
 
-    context 'when shows all the supervisions of tcc two calendar' do
-      it 'shows all the supervisions of tcc two with options' do
-        orientation = create(:current_orientation_tcc_two)
-        orientation.professor_supervisors << professor
+    it 'displays active link' do
+      expect(page).to have_selector("a[href='#{index_url}'].active")
+    end
+
+    it 'has links for details, activities, and documents' do
+      find('.academic-name-link').click
```

Comment:
Not use this, change for we did in the responsible orientation
Use the correct order:
```erb
          visit index_url

          orientation_link = "a[href='#{responsible_orientation_path(orientation)}']"
          find(orientation_link).click
```